.github/workflows/wheels.yml

```
@@ -0,0 +1,34 @@
+    name: Build wheels for ${{ matrix.python }}
+    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        python: ["3.9", "3.10", "3.11", "3.12", "3.13"]
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: "true"
+
+      - name: Set up Python ${{ matrix.PYTHON_VERSION }}
```

actionlint (1.7.4)

property "python_version" is not defined in object type {python: number}

(expression)